appveyor.yml

```diff
@@ -56,9 +56,10 @@ install:
   - cmd: conda create -n test python=%CONDA_PY%
   - cmd: activate test
   - cmd: conda update -q -y --all
-  - cmd: conda install -q -y -c conda-forge numpy scipy seaborn networkx pandas
-  - cmd: conda install -q -y -c conda-forge pytest pytables six numba='<0.47' sympy pyyaml
-  - cmd: conda install -q -y -c conda-forge psutil numexpr nodejs ipywidgets bokeh
+  - cmd: conda install -q -y numpy scipy seaborn networkx pandas
+  - cmd: conda install -q -y pytest pytables six sympy pyyaml
+  - cmd: conda install -q -y psutil numexpr nodejs ipywidgets bokeh
+  - cmd: conda install -q -y numba
```

Comment:
Seems to take care of versions to be newer in some cases. Not sure why conda-forge is doing weird things all of a sudden.